src/views/LogoutView.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import {Header, Image, Grid, Segment, Button, Icon} from 'semantic-ui-react';
+import logo from '../assets/logo.svg';
+import {ApolloConsumer} from 'react-apollo';
+import {auth} from '../state/auth';
+
+const LoginView = ({location, history}) => {
```

Comment:
You can remove the `return` and the `{}` here since you have no additional logic for the component.